Fix `starting container with HostConfig was deprecated since v1.10`

intval opened the pull request:
`starting container with HostConfig was deprecated since v1.10 and removed in v1.12`

by removing options that are regarded as `HostConfig` to the container creation section instead of container launching section

intval committed:
Fix `starting container with HostConfig was deprecated since v1.10 and removed in v1.12`
by removing options that are regarded as `HostConfig` to the container creation section instead of container launching section

mafintosh commented on index.js:
```diff
@@ -37,18 +37,19 @@ var run = function(image, opts) {
     Env: [],
-    Volumes: {}
+    Volumes: {},
+	HostConfig: HostConfig
```
could you fix the indentation here?

mafintosh commented:
added a minor comment. looks good otherwise!

intval committed:
Code format: indentation change

According to https://github.com/mafintosh/docker-run/pull/11#discussion_r88894631

mafintosh commented:
3.0.0 - thanks!

mafintosh commented:
Btw do you want to help maintain this project? I can add you as an owner if you want.

intval commented:
Sure, why not.
Thanks for your contributions so far as well.

2016-11-21 12:34 GMT-02:00 Mathias Buus <notifications@github.com>:

> Btw do you want to help maintain this project? I can add you as an owner
> if you want.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mafintosh/docker-run/pull/11#issuecomment-261954120>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA6pfmy2ZmzpmALGSrNbqx_0-BaKwnIiks5rAawMgaJpZM4KqipT>
> .
>
